src/etc/inc/interfaces.inc

```diff
@@ -2921,8 +2920,9 @@ function interface_dhcpv6_configure($interface = 'wan', $wancfg)
     /* Enable RFC6204w support for IPv6 Customer Edge (CE) router */
     set_single_sysctl("net.inet6.ip6.rfc6204w3", "1");
 
-    /* always kill rtsold in case of reconfigure */
+    /* always kill in case of reconfigure */
     killbypid("/var/run/rtsold_{$realifv6}.pid", 'TERM', true);
+    killbypid('/var/run/dhcp6c.pid', 'TERM', true);
```

Comment:
@marjohn56 maybe rtsold_script.sh should always fully reload as long as we can't SIGHUP interface listening. having it removed from the command line is the lesser problem in general

Reply:
Up to you. The option is there in the PR for dhcp6c if we decide to use it, if not can we just cherry pick the PDINFO bit, that'll be useful.

Reply:
rtsold/rtsold_script never comes from trackers, only from true WAN interfaces